ballista/executor/src/executor_server.rs

```diff
@@ -486,17 +497,23 @@ impl<T: 'static + AsLogicalPlan, U: 'static + AsExecutionPlan> ExecutorServer<T,
             task_execution_times,
         );
 
-        self.circuit_breaker_client
-            .deregister_scheduler(task_identity.to_owned())?;
+        if let Err(e) = self
+            .circuit_breaker_client
+            .deregister_scheduler(task_identity.to_owned())
+        {
+            error!(error = %e, task_identity, scheduler_id, "error de-registering circuit breaker");
+        }
 
         let task_status_sender = self.executor_env.tx_task_status.clone();
-        task_status_sender
+        if let Err(_e) = task_status_sender
             .send(CuratorTaskStatus {
                 scheduler_id,
                 task_status: vec![task_status],
             })
             .await
-            .unwrap();
+        {
+            error!("failed to send task status, channel closed");
+        }
```

Comment:
If the status receiver is closed then the executor should be in the process of shutting down anyway so we don't need to worry about this, just logging so we don't panic